doc/integrations/AWS_EC2/test.md

```diff
@@ -0,0 +1 @@
+
```

Comment:
This seems to be a new empty file unnecessarily added.

Reply:
Thanks, I removed it.